arch/arm/mach-bcm/Kconfig

```diff
@@ -160,6 +160,20 @@ config ARCH_BCM2835
 	  This enables support for the Broadcom BCM2835 and BCM2836 SoCs.
 	  This SoC is used in the Raspberry Pi and Roku 2 devices.
 
+config MACH_BCM2708
+	bool "Enable optimized __copy_to_user and __copy_from_user"
```

Comment:
I was tired and didn't know what to call it, so I just kept the name.
The only place CONFIG_MACH_BCM2708 is used is in the optimization code: https://github.com/raspberrypi/linux/commit/5fb515e14129b40fa8f3d8bf797d0a80b82fe512